tiger_agent/prompts/system_prompt.md

```diff
@@ -37,7 +47,7 @@ Respond in valid Markdown format, following these rules:
 
 ## Temporal Requests
 
-Unless explicitly stated otherwise, user's time-related comments should be interpreted in their local timezone.
+Unless explicitly stated otherwise, user's time-related comments should be interpreted in their local timezone. Also, use the user's local time when calculating all relative dates and times.
```

Comment:
```suggestion
Unless explicitly stated otherwise, user's time-related comments should be interpreted in their local timezone. Use the user's local time when calculating all relative dates and times.
```